# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/moz.build
# Commit: 81ddcb21f057
# Full Hash: 81ddcb21f05754f2b3b4e581e5d4295fada1cc2c
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-06-12 14:56:55
# Regressor Bug: 1641737
# File Overlap Count: 2
# Description:
#   Bug 1641737 - P2. Use MozPromise with DocumentLoadListener and remove cycle. r=mattwoodrow.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78179
# ==============================================================================

diff -r 02f45f660107 -r 81ddcb21f057 netwerk/ipc/moz.build
--- a/netwerk/ipc/moz.build	Fri Jun 12 05:56:26 2020 +0000
+++ b/netwerk/ipc/moz.build	Fri Jun 12 05:56:28 2020 +0000
@@ -5,7 +5,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 EXPORTS.mozilla.net += [
-    'ADocumentChannelBridge.h',
     'ChannelEventQueue.h',
     'DocumentChannel.h',
     'DocumentChannelChild.h',
